docs/providers/database.md

```diff
@@ -64,6 +64,17 @@ ClowdEnv Config options available:
 
 - `pvc`
 
+#### single
+
+(*EXPERIMENTAL*)
+
+In the single mode, the the **Database Provider** will provision a single node
+PostgreSQL instance, of latest supported version, for the whole ClowdEnvironment.
+Each ClowdApp requesting a database would get a separate user and
+a [PostgreSQL schema](https://www.postgresql.org/docs/current/ddl-schemas.html)
+with the same name as the user. The user has only access to create objects
+only within it's schema. Admin credentials are (currently) not provided.
```

Comment:
"it's" (abbreviation) -> "its" (possessive)  (nitpick :-) )

Reply:
fixed